logstash-core-event-java/src/test/java/com/logstash/EventTest.java

```diff
@@ -4,6 +4,9 @@
 
 import java.io.IOException;
 import java.util.*;
+
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.core.IsSame.sameInstance;
```

Comment:
I liked the `sameInstance` assertion from there, so I started typing it. and it worked. so I stuck with it